packages/indexer-common/src/allocations/query-fees.ts

```diff
@@ -39,8 +41,23 @@ export interface PartialVoucher {
   receipt_id_max: string // (0x-prefixed hex)
 }
 
+interface ReceiptMetrics {
+  receiptsToCollect: Gauge<string>
+  failedReceipts: Counter<string>
+  partialVouchersToExchange: Gauge<string>
+  receiptsCollectDuration: Histogram<string>
+  vouchers: Counter<string>
+  successVoucherRedeems: Counter<string>
+  invalidVoucherRedeems: Counter<string>
+  failedVoucherRedeems: Counter<string>
+  vouchersRedeemDuration: Histogram<string>
+  vouchersBatchRedeemSize: Gauge<never>
+  voucherCollectedFees: Gauge<string>
```

Comment:
I don't know much about Prometheus in TS, but why do we need those `string` and `never` as type parameters here?

Reply:
I assume they are the type signature for the labels, right?

Reply:
Yep, that's my understanding as well, that providing the types here allows us to tag the data with a label of that type later on